service/qos/service_level_controller.cc

```diff
@@ -407,6 +407,47 @@ future<> service_level_controller::update_cache(update_both_cache_levels update_
     }
 }
 
+service_level_options get_driver_service_level_slo() {
+    service_level_options slo;
+    slo.shares = 200;
+    slo.workload = service_level_options::workload_type::batch;
+    return slo;
+}
+
+future<utils::chunked_vector<mutation>> service_level_controller::get_create_driver_service_level_mutations(db::system_keyspace& sys_ks, api::timestamp_type timestamp) const {
+
+    utils::chunked_vector<mutation> mutations;
+
+    auto sys_ks_mutation = co_await sys_ks.make_service_level_driver_created_mutation(true, timestamp);
+    mutations.push_back(sys_ks_mutation);
+
+    auto [sl_mutations, _] = co_await raft_service_level_distributed_data_accessor::set_service_level_mutations(sys_ks.query_processor(), service_level_controller::driver_service_level_name, get_driver_service_level_slo(), timestamp);
+    std::move(sl_mutations.begin(), sl_mutations.end(), std::back_inserter(mutations));
+
+    co_return mutations;
+}
+
+future<> service_level_controller::create_driver_service_level(service::group0_guard guard, db::system_keyspace& sys_ks) {
```

Comment:
I'd call it: migrate_to_xxxx to show this is not the normal path, also shouldn't it reuse service_level_controller::get_create_driver_service_level_mutations